dev-python/rtslib-fb/rtslib-fb-2.1.73.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{6,7,8} )
```

Comment:
Can you add Python 3.9 next time around? Also, blank line after EAPI, and usually a blank line before inherit.